### PR TITLE
docs: prisma example

### PR DESCRIPTION
**Pull request by sophia-bq**
### Summary

Add an example of a Prisma application using the AWS Advanced Node.js Wrapper. 

### Description

A tutorial to set up a simple Prisma application which includes: 
- a database adapter that facilitates simple queries
- example project structure and set up 
- example code for running queries

### CODE REVIEW

File: examples/prisma_example/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    "aws-advanced-nodejs-wrapper": "^1.0.0",
+    "pg": "^8.13.1",
+    "util": "^0.12.5"
+  }
+}
+```
+
+## Step 3: Generate the data model.
+
+The data model can either be generated by reading it out from an existing database or by manually creating it. When it is manually created, it can be transferred to the database using Prisma Migrate.
```
Comment (crystall-bitquill) [inline examples/prisma_example/README.md:68]: What is Prisma Migrate? Is there documentation that can be linked available?

File: examples/prisma_example/README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  password: "password",
+  host: "db-identifier.XYZ.us-east-2.rds.amazonaws.com",
+  database: "postgres",
+  port: 5432
+});
+const adapter = new PrismaAws(client);
+const prisma = new PrismaClient({ adapter });
+```
+
+## Step 4: Query the database
```
Comment (crystall-bitquill) [inline examples/prisma_example/README.md:124]: Step 4 or 5?

File: examples/prisma_example/README.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+main();
+```
+
+The `client` is the AwsPgClient, and `prisma` the Prisma Client created in Step 3. You can now use the Prisma Client to make queries through the AwsPgClient via the adapter created in Step 3.
+
+Run the code with: `npx tsx src/index.ts`
+
+# Summary
+
+This tutorial walks through the steps required to add and configure the AWS Advanced Node.js Wrapper to a simple Prisma application.
```
Comment (crystall-bitquill) [inline examples/prisma_example/README.md:147]:
```suggestion
This tutorial walks through the steps required to add and configure the AWS Advanced NodeJS Wrapper to a simple Prisma application.
```

**Review by joyc-bq — APPROVED** [review]

File: examples/prisma_example/README.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+Before either option, edit `DATABASE_URL` in the `.env` file to be the connection string to your datasource.
+
+### Read it out from an existing database:
+
+- Remove any existing prisma/migrations directory, models from the Prisma schema file and migrations from the database
+- Run: `npx prisma db pull`.
+
+### Manually create a data model
+
+- Edit the `schema.prisma` file inside the prisma directory
```
Comment (crystall-bitquill) [inline examples/prisma_example/README.md:79]:
```suggestion
- Edit the `schema.prisma` file inside the `prisma` directory
```

File: examples/prisma_example/README.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  - An example can be found in schema.prisma in this tutorial.
+- Write the data model to your database run: `npx prisma migrate dev --name init`.
+
+You should see the data model in the prisma directory.
```
Comment (crystall-bitquill) [inline examples/prisma_example/README.md:83]:
```suggestion
You should see the data model in the `prisma` directory.
```

**Review by crystall-bitquill — APPROVED** [review]

File: examples/prisma_example/README.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    "util": "^0.12.5"
+  }
+}
+```
+
+## Step 3: Generate the data model.
+
+The data model can either be generated by reading it out from an existing database or by manually creating it. When it is manually created, it can be transferred to the database using [Prisma Migrate](https://www.prisma.io/docs/orm/prisma-migrate/getting-started).
+
+Before either option, edit `DATABASE_URL` in the `.env` file to be the connection string to your datasource.
```
[the remark this answers is not in the document]
Comment (sophia-bq) [reply examples/prisma_example/README.md:70]: Just for this step, afterwards we connect through the connection property specified in the AwsPgClient

File: examples/prisma_example/README.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
+# Tutorial: Getting Started with the AWS Advanced NodeJS Wrapper and Prisma ORM.
+
+In this tutorial, you will set up a Prisma application with the AWS Advanced NodeJS Wrapper, and use the Prisma client to execute a simple database operation.
+
+> [!NOTE] This tutorial was written for a PostgreSQL datasource using the following technologies:
```
Comment (karenc-bq) [inline examples/prisma_example/README.md:5]:
```suggestion
> [!NOTE]
> This tutorial was written for a PostgreSQL datasource using the following technologies:
```
Some formatting issues:
image

**Pull request merged**